pool.go

```diff
@@ -433,8 +481,22 @@ func (p *Pool) put(ioc *ioErrConn) {
 
 // Do implements the Do method of the Client interface by retrieving a Conn out
 // of the pool, calling Run on the given Action with it, and returning the Conn
-// to the pool
+// to the pool.
+//
+// If the given Action is a CmdAction, it will be pipelined with other concurrent
+// calls to Do, which can improve the performance and resource usage of the Redis
+// server, but will increase the latency for some of the Actions. To avoid the
+// automatic pipelining you can either set PoolPipelineWindow(0) when creating the
```

Comment:
`PoolPipelineWindow(0, 0)`* I swear this is the last thing and then this can go in >_<

Reply:
Done